scripts/comparator/docker-setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -xe
+
+REVISION=$(git rev-parse HEAD)
+COMPOSE_FILE=$GOPATH/src/github.com/m3db/m3/scripts/comparator/docker-compose.yml
+export REVISION
+
+function setup_docker {
+  echo "Run m3query, m3comparator, and prometheus containers"
+  docker-compose -f ${COMPOSE_FILE} up -d --build --renew-anon-volumes m3comparator
+  docker-compose -f ${COMPOSE_FILE} up -d --build --renew-anon-volumes prometheus
+  docker-compose -f ${COMPOSE_FILE} up -d --build --renew-anon-volumes m3query
+
+  CI=$1
+	if [[ "$CI" != "true" ]]
+  then
+    echo "run grafana container"
+    docker-compose -f ${COMPOSE_FILE} up -d --build --renew-anon-volumes grafana
+  fi
+}
+
+function teardown_docker {
+  CI=$1
+   # CI fails to stop all containers sometimes
+	if [[ "$CI" == "true" ]]
+  then
```

Comment:
nit: weird spacing here, mix of tabs and spaces perhaps?

Reply:
Ah yeah nice catch